src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,59 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import core.basesyntax.StorageObject;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+
+    private static final int MAX_ITEMS_NUMBER = 10;
+    private static final int PRIMARY_SIZE = 0;
+    private final StorageObject[] storage;
+    private int size;
+
+    public StorageImpl() {
+        this.size = PRIMARY_SIZE;
```

Comment:
redundant, java set 0 as default for primitive
```suggestion
    private final StorageObject[] storage;
    private int size;

    public StorageImpl() {
```